blaze-client/src/main/scala/org/http4s/client/blaze/package.scala

```diff
@@ -2,12 +2,14 @@ package org.http4s
 package client
 
 import cats.effect._
+import fs2.Stream
 
 package object blaze {
 
-  /** Default blaze client
-    *
-    * This client will create a new connection for every request.
-    */
-  def defaultClient[F[_]: Effect]: Client[F] = SimpleHttp1Client(BlazeClientConfig.defaultConfig)
+  def defaultClient[F[_]: Effect]: F[Client[F]] = Http1Client()
```

Comment:
What does this even get used for?

Reply:
Dunno, but with only one client it doesn't make much sense.

Reply:
Remove or leave with deprecation notice?

Reply:
I will always support deprecating changes that don't add drag for us to keep around.  Makes migration easier for users.